pkg/chezmoi/sourcestate.go

```diff
@@ -1371,6 +1380,55 @@ func (s *SourceState) getExternalData(
 		return nil, err
 	}
 
+	if external.Checksum.Size != 0 {
```

Comment:
Not a must-have, but it would be nice if the checksum process could run all of the present checksum values and then output all of the errors at once on output.

Reply:
Thanks, done.